code/java/api/openapi.yaml

```diff
@@ -1937,6 +1937,7 @@ components:
         perceelnummerVerschuiving:
           $ref: '#/components/schemas/TypePerceelnummerVerschuiving'
         adressen:
+          description: Alle binnenlandse adressen gekoppeld aan deze kadastraal onroerende zaak
```

Comment:
We hebben overal Markdown in de descriptions gebruikt. 
Als je hier het volgende van maakt is dit weer consistent met de rest van de API.

description: | 
                       Alle binnenlandse adressen gekoppeld aan deze kadastraal onroerende zaak

Ik heb hier spaties gebruikt voor het voorbeeld. Kijk even naar andere descriptions....

Reply:
Elders in dat bestand zie ik dat `|-` gebruikt wordt, zoals met dit voorbeeld:
```yaml
      description: |-
        Het bedrag dat een erfpachter moet betalen aan de eigenaar van een stuk grond omdat hij zijn grond gebruikt.

        Waardelijst in deze component : [soortErfpachtCanon](http://www.kadaster.nl/schemas/waardelijsten/SoortErfpachtcanon/)
```
Dus ik heb het aangepast:
```yaml
          description: |-
            Alle binnenlandse adressen gekoppeld aan deze kadastraal onroerende zaak
```
Overigens wordt dit lang niet overal consistent toegepast 😇